addons/prometheus-server/metadata.yaml

```diff
@@ -1,5 +1,5 @@
 name: prometheus-server
-version: v0.2.1
+version: v0.2.2-alpha.1
```

Comment:
why alpha version

Reply:
To prevent break changes affect existing GA version. These changes are in experimental now.

Reply:
fixed